examples/plot_dem_hillshade.py

```diff
@@ -0,0 +1,139 @@
+"""
+Create A Hillshade Layer Using EarthPy
+========================================
+
+Learn how to create a hillshade Layer from a DEM using EarthPy
+"""
+
+###############################################################################
+# Create A Hillshade Layer Using EarthPy
+# ---------------------------------------
+#
+# .. note::
+#       The examples below will show you how to use the ``es.hillshade()``
+#       function from EarthPy.
+
+###############################################################################
+# Create hillshade from DEM
+# -----------------------------
+# Creating a hillshade layer can be a great way to help visualize elevation
+# data, and make your map more presentable. This vignette will show you
+# how to create a hillshade layer using EarthPy, and the different ways to
+# modify the layer to get the hillshade to look exactly how you want it to.
+#
+# To begin using the EarthPy ``hillshade()`` function, import the needed
+# packages and create an array to be plotted.
+
+###############################################################################
+# Import Packages
+# ------------------------------
```

Comment:
The number of dashes here should be the same length as the header text "import packages". not longer